doc/INSTALL.md

```diff
@@ -66,7 +66,6 @@ Add the path where the library has been installed
 
 and run `ldconfig` as root.
 
-
```

Comment:
My first plan was to do so, but then I thought it seems out of place because no other plugin/tool/whatever is mentioned there (and I didn't notice I deleted one empty line accidentally until it was pushed, sorry).

Do you think a very general section like "Further Installation Manuals" would be fine?